packages/web-components/src/tabs/tabs.styles.ts

```diff
@@ -0,0 +1,250 @@
+
+  :host {
+    box-sizing: border-box;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    color: ${colorNeutralForeground2};
+    grid-template-columns: auto 1fr auto;
+    grid-template-rows: auto 1fr;
+  }
+
+  :host([disabled]) {
+    cursor: not-allowed;
+    color: ${colorNeutralForegroundDisabled};
+  }
+
+  :host([disabled]) ::slotted(fluent-tab) {
+    pointer-events: none;
+    cursor: not-allowed;
+    color: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) ::slotted(fluent-tab:after) {
+    background-color: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) ::slotted(fluent-tab[aria-selected='true'])::after {
+    background-color: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) ::slotted(fluent-tab:hover):before {
+    content: unset;
+  }
+
+  .tablist {
+    display: grid;
+    grid-template-rows: auto auto;
+    grid-template-columns: auto;
+    position: relative;
+    width: max-content;
+    align-self: end;
+    box-sizing: border-box;
+  }
+  ::slotted([slot='start']),
+  ::slotted([slot='end']) {
+    display: flex;
+    align-self: center;
+  }
+  ::slotted([slot='start']) {
+    margin-inline-end: 11px;
+  }
+  ::slotted([slot='end']) {
+    margin-inline-start: 11px;
+  }
+
+  .tabpanel {
+    grid-row: 2;
+    grid-column-start: 1;
+    grid-column-end: 4;
+    position: relative;
+  }
+  :host([orientation='vertical']) {
+    grid-template-rows: auto 1fr auto;
+    grid-template-columns: auto 1fr;
+  }
+  :host([orientation='vertical']) .tablist {
+    grid-row-start: 2;
+    grid-row-end: 2;
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: auto 1fr;
+    position: relative;
+    width: max-content;
+    justify-self: end;
+    align-self: flex-start;
+    width: 100%;
+  }
+  :host([orientation='vertical']) .tabpanel {
+    grid-column: 2;
+    grid-row-start: 1;
+    grid-row-end: 4;
+  }
+  :host([orientation='vertical']) ::slotted([slot='end']) {
+    grid-row: 3;
+  }
+
+  :host([appearance='subtle']) ::slotted(fluent-tab:hover) {
+    background-color: ${colorSubtleBackgroundHover};
+    color: ${colorNeutralForeground1Hover};
+  }
+
+  :host([appearance='subtle']) ::slotted(fluent-tab:active) {
+    background-color: ${colorSubtleBackgroundPressed};
+    fill: ${colorSubtleBackgroundPressed};
+    color: ${colorNeutralForeground1};
+  }
+
+  :host ::slotted(fluent-tab.animated)::after {
+    transition-property: transform;
+    transition-duration: ${durationSlow};
+    transition-timing-function: ${curveDecelerateMax};
+  }
+
+  :host([size='small']) ::slotted(fluent-tab) {
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    padding-top: ${spacingVerticalSNudge};
+    padding-bottom: ${spacingVerticalSNudge};
+    padding-left: ${spacingHorizontalSNudge};
+    padding-right: ${spacingHorizontalSNudge};
+    border-radius: ${borderRadiusMedium};
```

Comment:
If border-radius is consistent across all, can we reduce to just the primary selector all slotted tabs rather than duplicating between small/large (both are the same)